src/api/login/login.ts

```diff
@@ -28,26 +31,34 @@ router.get('/', verifyToken, (req, res) => {
 
 router.get('/secure', verifyToken, (req, res) => {
     jwt.verify(req.token, 'secretKeyHere', (err, authData) => {
-        if(err) {
-            res.sendStatus(403);
-        } else {
-            res.json({
-                message: 'Secure Route',
-                authData: authData,
-            });
+        try{
+            if(err) {
+                throw new BadRequestError('Not allowed.');
+            } else {
+                res.json({
+                    message: 'Secure Route',
+                    authData: authData,
+                });
+            }
+        } catch(e) {
+            res.send(e)
         }
     });
 });
 
 router.post('/secure', verifyToken, (req, res) => {
     jwt.verify(req.token, 'secretKeyHere', (err, authData) => {
-        if(err) {
-            res.sendStatus(403);
-        } else {
-            res.json({
-                message: 'Secure Route',
-                authData: authData,
-            });
+        try {
+            if(err) {
+                throw new BadRequestError('Not allowed.');
+            } else {
+                res.json({
+                    message: 'Secure Route',
+                    authData: authData,
+                });
+            }
+        } catch(e) {
+            res.send(e)
```

Comment:
Why are you calling the middleware `verifyToken` AND verifying token on the route?